src/pages/[userId]/prospects.js

```diff
@@ -23,52 +66,11 @@ export default function Prospects() {
     isLoading: isLoadingUser,
     error: userError,
   } = getMinisterById(`${userId}`);
```

Comment:
Is there a reason this is in a template literal?

Reply:
Lol. Remember how I mentioned the api call url was broken? This was a hail mary before I found the solution. I didn't think it would work, but I was trying anything and everything at that point. I just fixed it.